test/index.js

```diff
@@ -227,11 +227,14 @@ describe('compose()', function () {
             server.select('a').inject('/a/plugin', function (response) {
 
                 expect(response.statusCode).to.equal(200);
+
+                /* eslint-disable hapi/no-shadow-relaxed */
```

Comment:
I would go with `responseA` and `responseB` or something rather than disabling the rule